tests/testthat/test-utils-parse.R

```diff
@@ -359,7 +359,7 @@ test_that('parseExcludedControlConditions properly retrieves the excluded contro
   
   conditions <- repgen:::parseExcludedControlConditions(controlConditionJSON)
   
-  expect_equal(repgen:::parseExcludedControlConditions(c()), NULL)
+  expect_equal(repgen:::parseExcludedControlConditions(), NULL)
```

Comment:
ah, figured this out. when I changed the utils-read function readExcludedControlConditions() to not use validation since the new excludeConditions data have no required fields, the function returns something other than NULL. I changed so that I still use the validateFetchedData function (with no required fields) and this test can be changed back. 😃